plugins/aerospike.yaml

```diff
@@ -69,10 +69,30 @@ pipeline:
       preserve_to: aerospike_severity
       mapping:
         info: detail
+    output: config_filter
+
+  - id: config_filter
+    type: filter
+    expr: '$record.source_file == "cfg.c" && $record.source_location == "4058"'
```

Comment:
4058 is very specific. Are we sure we want that?

Reply:
I was concerned with this as well. I will filter based on context config.